docs/toggles/application_error_report.rst

```diff
@@ -0,0 +1,17 @@
+Application Error Report
+========================
+This plugin is a useful tool for debugging errors that are preventing mobile users from
+successfully submitting a form. This includes things such as:
+- A calculation error related to a function that CommCare does not recognize being used
+in the calculation.
+- Errors related to trying to access data using the incorrect path.
```

Comment:
So, this is the one that is reported to sumologic instead yes?
Good to keep a note that for dimagi maintained environments this is sent to Sumologic so that its clear for core maintainers as well.

Reply:
[cd542f8](https://github.com/dimagi/commcare-hq/pull/34548/commits/cd542f828e9e5cedf8f9848fdda61ba89c1e3d61)